website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -107,31 +64,41 @@ All Toasts have icons by default that are intentionally tied to the Toast color.
 
 Icons within `neutral` and `highlight` Toasts can be replaced with other icons. Change them only when the new icon provides the user with extra value; otherwise, use the default icon provided.
 
-<Doc::Layout @spacing="12px" @direction="vertical">
-  <Hds::Toast @color="neutral" @icon="running" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Plan running</T.Title>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-  <Hds::Toast @color="success" @icon="check-circle" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Plan finished</T.Title>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-</Doc::Layout>
+![Two toasts, one neutral with a non-standard icon and one success with a standard icon.](/assets/components/toast/toast-icon-neutral-and-success.png)
 
 ## Size
 
-Toasts can be sized between 360px and 500px wide. Anything wider than 500px will lead to implementation issues and take up too much of the screen space. 
+Toasts can be sized between 360px and 500px wide. Anything wider than 500px is considered disruptive to the user experience and may cover important information on the page.
 
 ## Placement
 
-Toasts should appear in the bottom right corner of the screen with a margin of 32px from the bottom and a 24px from the right side of the viewport. 
+Toasts appear in the bottom right corner of the viewport with a margin of 32px from the bottom and 24px from the right side of the viewport. If a warning or error is contextual to a specific UI feature, such as a form, directly associate an Alert to that feature instead of a Toast.
 
-![Toast placement example](/assets/components/toast/toast-placement.png =600x*)
+![](/assets/components/toast/toast-placement.png =600x*)
 
 ### Displaying multiple Toasts
```

Comment:
Do we have any guidance on how many Toasts are allowed on a page or should we note that more than 2-3 Toasts isn't an ideal user experience?

Reply:
I think we don't necessarily have guidance for that because theoretically they can keep climbing. @jorytindall you have thoughts on this?

Reply:
No we don't have any guidance on the specific number of toasts. For example, I think based on how cloud-ui has the display of the Toast configured, they'll just continue to render as a stack and eventually extend past the top of the viewport. Maybe something for us to think about, but I also might note that this is a product-specific problem. Maybe it would be useful to include something _very_ general, but I don't think we generally run into this very frequently unless there's some sort of error (at which point it's unintentional to show many toasts) like something being triggered on a loop and showing notifications constantly.

Just a couple of thoughts